ios/FluentUI/Vnext/List/List.swift

```diff
@@ -68,6 +68,21 @@ public struct MSFListView: View {
                 }
             }
     }
+
+    private func dividers(_ sections: [MSFListSectionState]) -> [MSFListSectionState] {
```

Comment:
What would be a more meaningful name for this function? Just "dividers" seems unclear to me.